### ISSUE TITLE
Issues regarding the Flow tutorials

### ISSUE THREAD

**Issue by weizi-li** [flow-project/flow#680]
First of all, thank you for putting up all the tutorials! They are very helpful and I learned a lot from them. 

Last week, I've started to learn these tutorials and found some aspects of the tutorials can be potentially improved. These aspects range from general suggestions and things I feel aren't very clear. I've also fixed some typos and made some edits, which will show in the pull request. In the following, I'll elaborate things that aren't included in the pull request.

Some general suggestions first:
1. I feel like the whole tutorial series should include (or at least point to) a tutorial on basic RL, nothing fancy just explain the terms. I would imagine most folks interested in Flow come from either transportation or CS background. Neither group, especially the transportation folks, is necessarily know RL. They may have heared that RL can be used to play Atari games at the human level but probably aren't familiar with "state/action/observation space", "rollout", "MDP/POMDP", and "reward function", which terms have already spread out in tutorial01. These terms can be intimidating and the lack of explanation of them can be off-putting for folks want to use Flow.

2. In some tutorials, the current way of arranging code confuses me. When I see code blocks get introduced one after the other, I assume by copying all of them to a stand-alone file and by executing the file, I will get all results. However, this is not the case. For example,  in tutorial01_sumo.ipynb, if I do this, I can't see the CSV file reading results for some reason. Maybe because I did something wrong, if not, I would appreciate a solution to this issue.

3. There are several places across different tutorials mentioning "exercise0x". Should they be changed to "tutorial0x"?

4. in vehicles.add() across different tutorials, sometimes the "veh_id" is omitted sometimes not, be consistent?

Here comes the specifics:
Tutorial 01 SUMO
"custom environments may be used to modify the dynamical features of an scenario."
What are the dynamical features? If they will not be discussed in this tutorial, I suggest delete this sentence. 

"merges in both closed and open settings"
This is also confusing. Be more specific? 

"proper network features/behavior"
What you mean by network behavior?

"in the scenario's documentation"
give a link here?

"emissions output"
confusing? maybe reword it to something like "a summary of the simulation"?

Tutorial 06 Scenarios
2. Multiple routes per edges:
After the first paragraph, give an example actually has multiple routes with their probabilities sum up to 1?

From "In all three cases..." until "2. Specifying Auxiliary Scenario Features"
I don't understand the message here.

"Finally, we are able to visually confirm..."
Give a screenshot to show what users should expect?

Tutorial 10 Controllers
2.1 BaseLaneChangeController and 2.2 Lane-Change Command
These two subsections are probably the most confusing to me. I am totally lost among "move a vehicle into lane 2", "Lane 0 is the farthest-right, so the direction -1 is a lane change to the right.", and "In that case, a lane-change to the right is specified by returning the direction -1. If the vehicle is in lane 0, then the direction 0 is returned."

Tutorial 12 Controllers
"cause collisions with vehicles that are manually added into the network"
"being manually pre-inserted into the network"
Are the manually added vehicles different from the vehicles specified in inflows? What are the manually added vehicles? 

"depart_speed="speedLimit",
For some reason, this will not compile. Change "speedLimit" to "random" solves the issue. 
Error message in SUMO:
Error: value 'speedLimit' does not match any member types of the union
My environment:
OS: Ubuntu 16.04
SUMO: Eclipse SUMO Version v1_1_0+0000-2147d15
PYTHON: 3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34) 
[GCC 7.3.0]

**Comment by ashkan-software** [comment]
Thank you very much @weizi-li for your time and writing such long and detailed description of your suggestions. We certainly take all of your suggestions into consideration. :)

**Comment by eugenevinitsky** [comment]
Hi, I really do like the short MDP/RL tutorial suggestion. I think we will port one in.

**Comment by xiaoToby** [comment]
Yeah, I also agree with the MDP/RL tutorial suggestion.
Because I also confront an issue that I want the routes to be random.
On sumo, there is an assistant file (randomTrip.py) to help to create random routes of a network,
but it cannot work on FLOW.
I think random routes function has a big influence on some project which based on FLOW.
